scala/src/main/scala/com/scalableminds/webknossos/wrap/WKWFile.scala

```diff
@@ -100,7 +100,7 @@ trait WKWCompressionHelper extends BoxImplicits {
   }
 }
 
-class WKWFile(val header: WKWHeader, fileMode: FileMode.Value, underlyingFile: RandomAccessFile) extends WKWCompressionHelper with WKWMortonHelper {
+class WKWFile(val header: WKWHeader, fileMode: FileMode.Value, underlyingFile: RandomAccessFile, underlyingFilePath: String) extends WKWCompressionHelper with WKWMortonHelper {
```

Comment:
I store `underlyingFilePath` with the WKWFile wrapper because the `RandomAccessFile` API doesn’t expose the file path that was passed to it. Previously we (unsafely) accessed its private path member, which also broke JRE 11+ compatibility